src/bespokelabs/curator/request_processor/openai_batch_request_processor.py

```diff
@@ -459,33 +483,32 @@ async def check_batch_status(self, batch_id: str) -> Batch | None:
 
     async def watch(self) -> None:
         """Monitor the status of batches until all are completed (includes successfully, failed, expired or cancelled)."""
-        # progress bar for completed requests
-        pbar = tqdm(
-            total=self.tracker.n_submitted_requests,
-            desc="Completed OpenAI requests in batches",
-            unit="request",
-        )
-
         # resume from already downloaded batches
         response_files_found = 0
         all_response_files = set(glob.glob(f"{self.working_dir}/responses_*.jsonl"))
         for batch_id, request_file in self.batch_id_to_request_file_name.items():
-            request_file_idx = request_file.split("/")[-1].split("_", 1)[1]
-            response_file = f"{self.working_dir}/responses_{request_file_idx}"
+            response_file = request_file_to_response_file(request_file, self.working_dir)
             if response_file in all_response_files:
-                logger.info(
+                logger.debug(
```

Comment:
i think we should initialize the pbar before this code and update how many responses are already downloaded in existing response files. otherwise, we get this confusing logging behavior where we never track the finished responses. example log


```
2024-12-04 14:55:30,187 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - 1 out of 3 batches already downloaded.
2024-12-04 14:55:30,187 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - 2 out of 2 remaining batches are already submitted.
Completed OpenAI requests in batches:   0%|                                                              | 0/3 [00:00<?, ?request/s]2024-12-04 14:55:47,141 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - Batch batch_67506b30893481918d7c660687c5f006 returned with status: completed
2024-12-04 14:55:47,157 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - Batch batch_67506b306648819194c2e98de389b103 returned with status: completed
Completed OpenAI requests in batches:  67%|████████████████████████████████████                  | 2/3 [00:00<00:00,  9.80request/s]2024-12-04 14:55:47,458 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - Batch batch_67506b306648819194c2e98de389b103 completed and downloaded
2024-12-04 14:55:47,985 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - Batch batch_67506b306648819194c2e98de389b103 written to /home/trung/.cache/curator/47f1b0209c82f62b/responses_2.jsonl
2024-12-04 14:55:47,988 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - Batch batch_67506b30893481918d7c660687c5f006 completed and downloaded
2024-12-04 14:55:47,990 - bespokelabs.curator.request_processor.openai_batch_request_processor - INFO - Batch batch_67506b30893481918d7c660687c5f006 written to /home/trung/.cache/curator/47f1b0209c82f62b/responses_1.jsonl
Completed OpenAI requests in batches:  67%|████████████████████████████████████                  | 2/3 [00:01<00:00,  1.09request/s]
```

Note how one batch is already downloaded, but the completed responses from that batch are not factored into the pbar, so we start from 0. I would expect pbar to start from 1 given 1 batch already finished ---- the user might think our caching is broken with this behavior.

Reply:
This was solved in a separate PR :) 
#197